content/content.json

```diff
@@ -162,13 +162,20 @@
       "title": "Using .goosehints Files with Goose",
       "link": "https://dev.to/lymah/using-goosehints-files-with-goose-304m",
       "contributor": "Lymah123"
-    }, 
+    },
     {
       "type": "blog",
       "thumbnail": "./img/using-screentoolkit-with-goose.png",
       "title": "Using the Screen Tool kit with goose",
       "link": "https://dev.to/iabdulkarim472/using-the-screen-tool-kit-with-goose-3916",
       "contributor": "Ibrahim Abdulkarim"
+    },
+    {
+      "type": "social post",
+      "thumbnail": "./img/Social-post-how-are-you-using-Goose.png",
+      "title": "How are you using Goose?",
+      "link": "https://www.threads.net/@lymahcodes/post/DCmfASnMvWq?xmt=AQGzNfUQKe2o5zicl8zm6D13opuEIwsatcGmAknzYKCBTg",
```

Comment:
Love the use case of using Goose to analyze your emails! Thanks for the contribution!